### ISSUE TITLE
NPE when getting the commit change log in Git

### ISSUE THREAD

**Issue by dcoraboeuf** [nemerosa/ontrack#376]
In version 2.17.4:

```
java.lang.NullPointerException
	at org.eclipse.jgit.lib.ObjectIdOwnerMap.get(ObjectIdOwnerMap.java:131)
	at org.eclipse.jgit.revwalk.RevWalk.parseAny(RevWalk.java:837)
	at org.eclipse.jgit.revwalk.RevWalk.parseCommit(RevWalk.java:752)
	at net.nemerosa.ontrack.git.support.GitRepositoryClientImpl.range(GitRepositoryClientImpl.java:477)
	at net.nemerosa.ontrack.git.support.GitRepositoryClientImpl.graph(GitRepositoryClientImpl.java:177)
	at net.nemerosa.ontrack.extension.git.service.GitServiceImpl.getChangeLogCommits(GitServiceImpl.java:232)
```

**Comment by dcoraboeuf** [comment]
Repository declared in the project properties was not synched (indexation = 0). Therefore, the commits were not available at all.

Consider forcing the sync upon change log computation (when fetching the commits).

**Comment by dcoraboeuf** [comment]
No force the sync but return a meaningful message:

* if the sync is currently running
* if the sync has never been done

**Comment by dcoraboeuf** [comment]
Linked to #377

**Comment by dcoraboeuf** [comment]
Fixed by #377
